### PR TITLE
Updates to Password Change doc

### PR DESCRIPTION
**Pull request by jeffreylees**
https://docs-content-staging-pr-8213.herokuapp.com/docs/connections/database/password-change

### CODE REVIEW

File: articles/connections/database/password-change.md [hunk 1]
```diff
@@ -99,32 +107,18 @@ The Management API v2 provides an additional endpoint, [Generate a password rese
...
-![](/media/articles/connections/database/lock_v9/lock_request_reset.png)
-
-:::panel-warning Notice
-If you are using Lock version 8, the user will be asked, immediately after clicking the **Don't remember your password?** link on the Lock screen, to provide their email address and their new password. The user would then confirm this action via email.
-
-However, this flow is not considered safe. We recommend that you upgrade to Lock 9 or later to utilize a more secure flow. To learn more about migrating to Lock, see [Vulnerable Password Flow](/migrations/past-migrations#vulnerable-password-flow).
-:::
-
-The user will receive an email containing a link to reset the password, and the flow continues exactly as if the [Authentication API method](#using-the-authentication-api) was used.
+However, for applications using the New Universal Login Experience, the redirect destination is handled differently. Users are instead redirected to the [default login route](/universal-login/default-login-url) upon success. Any error cases are then automatically handled as part of the Universal Login flow, and the Redirect URL in the email template will be ignored.
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]: I think we are already mentioning this above

File: articles/connections/database/password-change.md [hunk 2]
```diff
@@ -12,36 +12,39 @@ useCase: customize-connections
...
+- [**Authentication API**](#use-the-authentication-api): Send a `POST` call to the Authentication API to send a password reset email to the user.
+- [**Embedded Lock**](#use-lock): The user uses the Lock login screen embedded in their app to trigger a password reset email.
+
+### Use Universal Login
 
-+ [**Authentication API**](#using-the-authentication-api): Send a `POST` call to the Authentication API to send a password reset email to the user.
-+ [**Lock**](#using-lock): The user uses the Lock login screen to trigger a password reset email.
+If your application is using Universal Login, the user will be able to trigger a password reset from the login page. With the New Universal Login Experience, the user will click the **Don't remember your password?** link and then enter their email address.
+
+::: note
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]: The experience is basically the same for the new and old UL, do we need to talk about both?

File: articles/connections/database/password-change.md [hunk 3]
```diff
@@ -65,18 +68,25 @@ If you're calling this from the browser, don't forget to add your URL to the `Al
 If your Connection is a custom database and the user exists in the database, invoke the Authentication API for `changePassword`.
 :::
 
-If the `POST` call is successful, the user receives an email containing a link to reset their password.
+Once the password reset has been triggered, the user will now [receive a password reset email](#password-reset-emails).
 
-![](/media/articles/connections/database/reset-password-email.png)
+### Use Lock
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]: Do we need to mention Lock here? In the context of avoiding talking about Lock / Embedded login, unless it's the specific Embedded Login section, I'd remove it.

File: articles/connections/database/password-change.md [hunk 4]
```diff
@@ -65,18 +68,25 @@ If you're calling this from the browser, don't forget to add your URL to the `Al
...
 
-![](/media/articles/connections/database/reset-password.png)
+The user clicks on the **Don't remember your password?** link on the Lock screen, and then enters their email address.
 
-After submitting the new password, confirmation that the user will be able to login with their new credentials appears:
+Once the password reset has been triggered, the user will now receive a password reset email.
+
+## Password reset emails
+
+Regardless of how it was triggered, if the `POST` call is successful, the user receives an email containing a link to reset their password.
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]: I'd remove "If the POST call is successful"

File: articles/connections/database/password-change.md [hunk 5]
```diff
@@ -65,18 +68,25 @@ If you're calling this from the browser, don't forget to add your URL to the `Al
...
 
-![](/media/articles/connections/database/lock_v9/lock_pass_changed.png)
+![](/media/articles/connections/database/password-reset-email.png)
 
+Clicking the link will send the user to the customizable [password reset page](/universal-login/password-reset).
+
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md:75]: I'd remove 'customizable'

File: articles/connections/database/password-change.md [hunk 6]
```diff
@@ -99,32 +107,14 @@ The Management API v2 provides an additional endpoint, [Generate a password rese
...
-
-:::panel-warning Notice
-If you are using Lock version 8, the user will be asked, immediately after clicking the **Don't remember your password?** link on the Lock screen, to provide their email address and their new password. The user would then confirm this action via email.
-
-However, this flow is not considered safe. We recommend that you upgrade to Lock 9 or later to utilize a more secure flow. To learn more about migrating to Lock, see [Vulnerable Password Flow](/migrations/past-migrations#vulnerable-password-flow).
-:::
-
-The user will receive an email containing a link to reset the password, and the flow continues exactly as if the [Authentication API method](#using-the-authentication-api) was used.
+In the Classic Experience you can [configure a url](/email/templates#configuring-the-redirect-to-url) to redirect users after completing the password reset. The URL will receive a success indicator and a message. The New Experience will redirect the users to the [default login route](/universal-login/default-login-url) when it succeeds, and will handle the error cases as part of the Universal Login flow. The Redirect URL in the email template will be ignored.
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]:
```suggestion
In the [Classic Universal Login Experience](/universal-login/classic) you can [configure a url](/email/templates#configuring-the-redirect-to-url) to redirect users after completing the password reset. The URL will receive a success indicator and a message. The [New Experience](/universal-login/new) will redirect the users to the [default login route](/universal-login/default-login-url) when it succeeds, and will handle the error cases as part of the Universal Login flow. The Redirect URL in the email template will be ignored.  
```

File: articles/connections/database/password-change.md [hunk 7]
```diff
@@ -158,14 +148,11 @@ Users will not receive notification that their password has been manually change
...
+1. Enter the new password, and click **Save**.
 
-2. Click on the **Actions** button on the right side of the page, and select **Change Password**.
-
-![](/media/articles/connections/database/manual-password-change.png)
-
-3. Enter the new password, and click **Save**.
-
-## Change Password Expiration Settings using Rules
+## Change password expiration settings using Rules
```
Comment (aaguiarz) [inline articles/connections/database/password-change.md]: I think this does not belong to this document

**Review by aaguiarz — DISMISSED** [review]
More comments..

File: articles/connections/database/password-change.md [hunk 8]
```diff
@@ -12,36 +12,39 @@ useCase: customize-connections
...
-
-To determine the flow you are using, navigate to [Dashboard > Tenant Settings > Advanced](${manage_url}/#/tenant/advanced) to check if the **Change Password flow v2** toggle is enabled. If it is, use Lock 9+. If not, use an older version of Lock to trigger the old Change Password flow.
-
-We strongly encourage you to enable **Change Password flow v2**. To learn more about the vulnerability and migration, please see [Vulnerable Password Flow](/migrations/past-migrations#vulnerable-password-flow). To learn more about migrating to Lock 11, please take a look at the [Lock 11 Migration Guide](/libraries/lock/v11/migration-guide).
+This article will help you learn how to reset a user's password. If you are trying to configure the custom Password Reset page for your tenant, see [Password Reset Page](/universal-login/password-reset). If you are a user of Auth0 and are trying to reset the password to your Auth0 account, see [Reset Your Auth0 Account Password](/support/reset-account-password).
```
Comment (erjohnson) [inline articles/connections/database/password-change.md]:
```suggestion
This article covers how to reset a user's password. If you are trying to configure the custom Password Reset page for your tenant, see [Password Reset Page](/universal-login/password-reset). If you are a user of Auth0 and are trying to reset the password to your Auth0 account, see [Reset Your Auth0 Account Password](/support/reset-account-password).
```

File: articles/connections/database/password-change.md [hunk 9]
```diff
@@ -12,36 +12,39 @@ useCase: customize-connections
...
+::: note
+If your application uses the Classic Universal Login Experience, it by default uses the Lock Widget, which also has a forgot password link, which brings up a screen that will allow the user to enter their email address much in the same way.
+:::
+
+This will fire off a POST request to Auth0 that will trigger the password reset process. Once the password reset has been triggered, the user will now [receive a password reset email](#password-reset-emails).
 
 ### Use the Authentication API
 
-To start an interactive password reset flow using the Authentication API, make a `POST` call specifying the email address of the user account whose password you would like to reset in the `email` field. If the call is successful, the user will receive an email prompting them to change their password.
+If your application uses start an interactive password reset flow using the Authentication API, make a `POST` call specifying the email address of the user account whose password you would like to reset in the `email` field. If the call is successful, the user will receive an email prompting them to change their password.
```
Comment (erjohnson) [inline articles/connections/database/password-change.md]: If your application uses ~start~?

**Review by erjohnson — APPROVED** [review]

**Comment by khorne3** [comment]
@jeffreylees Good to go, or should I wait?

**Comment by jeffreylees** [comment]
Good to go

**Comment by khorne3** [comment]
@aaguiarz Can you please review and (if appropriate) approve? I can merge at that point.

**Review by khorne3 — APPROVED** [review]

**Pull request merged**